src/Maker/MakeStimulusController.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Maker;
+
+use Symfony\Bundle\MakerBundle\ConsoleStyle;
+use Symfony\Bundle\MakerBundle\DependencyBuilder;
+use Symfony\Bundle\MakerBundle\Generator;
+use Symfony\Bundle\MakerBundle\InputConfiguration;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\WebpackEncoreBundle\WebpackEncoreBundle;
+
+/**
+ * @author Abdelilah Jabri <jbrabdelilah@gmail.com>
+ *
+ * @internal
+ */
+final class MakeStimulusController extends AbstractMaker
+{
+    private const SUFFIX = '_controller.js';
+
+    private const PATH = 'assets/controllers/';
+
+    public static function getCommandName(): string
+    {
+        return 'make:stimulus-controller';
+    }
+
+    public static function getCommandDescription(): string
+    {
+        return 'Creates a new Stimulus controller';
+    }
+
+    public function configureCommand(Command $command, InputConfiguration $inputConfig)
+    {
+        $command
+            ->addArgument('name', InputArgument::OPTIONAL, 'The name of the stimulus controller (e.g. <fg=yellow>hello</>)')
+            ->setHelp(file_get_contents(__DIR__.'/../Resources/help/MakeStimulusController.txt'));
+    }
+
+    public function generate(InputInterface $input, ConsoleStyle $io, Generator $generator)
+    {
+        $controllerName = $input->getArgument('name');
+        $fileName = $controllerName.self::SUFFIX;
+        $filePath = self::PATH.$fileName;
```

Comment:
Same for this one as well.
```suggestion
        $filePath = sprintf('assets/controllers/%s', $fileName);
```